dts/bindings/usb/gd,gd32-otgfs.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2017, I-SENSE group of ICCS
+# SPDX-License-Identifier: Apache-2.0
+
+description: GD32 OTGFS controller
+
+compatible: "gd,gd32-otgfs"
+
+include: [usb-ep.yaml, pinctrl-device.yaml, reset-device.yaml]
+
+properties:
+  reg:
+    required: true
+
+  interrupts:
+    required: true
+
+  pinctrl-0:
+    required: true
+
+  pinctrl-names:
+    required: true
+
+  ram-size:
+    type: int
+    required: true
+    description: |
+      Size of USB dedicated RAM. GD32 SOC's reference
+      manual defines a shared FIFO size.
+
```

Comment:
Try use up to 80 cols always possible.
You must add devicetree documentation here with examples. Each non standard property should be very well documented in a way that devicetree maintainers don't need look in the datasheet.